cmd/samples/samples.go

```diff
@@ -137,8 +138,15 @@ func (s *samples) loadConfig() error {
 	s.Env = s.ServerConfig.Tenant.EnvName
 	s.Namespace = s.ServerConfig.Global.Namespace
 
+	// handle configs for analytics-related credential
 	if s.ServerConfig.IsGCPManaged() {
-		s.EncodedName = provision.EnvScopeEncodedName(s.Org, s.Env)
+		s.IsGCPManaged = true
+		// SA credentials supersede the fluentd enpoint
```

Comment:
Should we log a warning if both exist and the fluentd endpoint is being superceded?

Reply:
Sounds good.